### PR TITLE
feat: Use appearance setter available via XCUIDevice

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Looks like Apple added some new handy stuff there since XCode 13

### CODE REVIEW

**Review by jlipps — APPROVED** [review]
cool!

**Review by KazuCocoa — APPROVED** [review]
great to find. This worked against a real device as well

File: WebDriverAgentLib/Commands/FBCustomCommands.m [hunk 1]
```diff
@@ -369,6 +370,25 @@ + (NSDictionary *)processArguments:(XCUIApplication *)app
...
 
++ (id<FBResponsePayload>)handleSetDeviceAppearance:(FBRouteRequest *)request
+{
+  NSString *name = [request.arguments[@"name"] lowercaseString];
+  if (nil == name || !([name isEqualToString:@"light"] || [name isEqualToString:@"dark"])) {
+    NSString *message = @"The appearance name must be either 'light' or 'dark'";
+    return FBResponseWithStatus([FBCommandStatus invalidArgumentErrorWithMessage:message traceback:nil]);
+  }
+
+  FBUIInterfaceAppearance appearance = [name isEqualToString:@"light"]
```
Comment (KazuCocoa) [inline WebDriverAgentLib/Commands/FBCustomCommands.m:381]: `0` seems automatic.
I've checked it via `appearanceMode`

File: PrivateHeaders/XCTest/XCUIDevice.h [hunk 2]
```diff
@@ -11,9 +11,16 @@
...
-@property(readonly) id screenDataSource; // @synthesize screenDataSource=_screenDataSource;
+@property (readonly) id screenDataSource; // @synthesize screenDataSource=_screenDataSource;
+
 - (_Bool)performDeviceEvent:(id)arg1 error:(id *)arg2;
 
+// Since Xcode 13
+// 1 - Light
+// 2 - Dark
+- (void)setAppearanceMode:(long long)arg1;
+- (long long)appearanceMode;
```
Comment (KazuCocoa) [inline PrivateHeaders/XCTest/XCUIDevice.h:22]: It seems like `userInterfaceStyle` in device info should follow this value if available.
In a real device, this value is 0 -> automatic, 1 -> light, 2 -> dark.
(Not for this pr itself)

**Pull request merged**